.github/workflows/release.yml

```diff
@@ -289,6 +289,28 @@ jobs:
           asset_name: saucetpl.tar.gz
           asset_content_type: application/tar+gzip
 
+  # Post Deploy Tests
+  post-release-windows-tests:
+    runs-on: ubuntu-latest
+    needs: release-windows-bundle
+    env:
+      SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+      SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Deps
+        run: npm ci
+      - name: saucectl test
+        working-directory: ./tests/fixtures/devxpress-test
+        # TODO: Add runner_version as a parameter so that it can pull from github releases
+        run: |
+          VERSION=$(curl -s -H "Authorization: token ${{ github.token }}" \
+            https://api.github.com/repos/${{ github.repository }}/releases | \
+            jq -r "[.[] | select(.draft == true) | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .tag_name")
+          echo Version:
+          echo $VERSION
+          npx saucectl run --test-env sauce --ccy 5 --region us-west-1 --runner-version "github-release: $VERSION"
```

Comment:
Since the saucectl has been removed from runner, it'd better to download it in preparation stage and then run that. I think npm install or download binary both are fine. 

Binary example: https://github.com/saucelabs/testrunner-toolkit/blob/master/.github/workflows/tests.yml#L76

Reply:
`devDependencies` (see previous comment)

Reply:
This will fail when loading the runner. At this time, the release is a draft, meaning, it will not be visible by users that don't have write access to the repo. Chef won't be able to get the attached runner to the draft release.

It requires chef to be authenticated, or do the post-deploy test *after* the `publish-release` step.